<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TBBuddyMainWindow</class>
 <widget class="QMainWindow" name="TBBuddyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>608</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="TBQt.qrc">
    <normaloff>:/Resources/icon_256x256.png</normaloff>:/Resources/icon_256x256.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="leftTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="centerTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="rightTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTournament">
    <property name="title">
     <string>Tournament</string>
    </property>
    <addaction name="actionQuickStart"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="actionConfigure"/>
    <addaction name="actionAuthorize"/>
    <addaction name="actionPlan"/>
    <addaction name="separator"/>
    <addaction name="actionShowHideMainDisplay"/>
    <addaction name="actionShowMoves"/>
    <addaction name="actionRebalance"/>
    <addaction name="separator"/>
    <addaction name="actionPauseResume"/>
    <addaction name="actionPreviousRound"/>
    <addaction name="actionNextRound"/>
    <addaction name="actionCallClock"/>
    <addaction name="actionEndGame"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowHideMainDisplay"/>
    <addaction name="actionShowHideSeatingChart"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Poker_Buddy"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTournament"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuickStart"/>
   <addaction name="actionAuthorize"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionPlan"/>
   <addaction name="actionExport"/>
   <addaction name="actionRebalance"/>
   <addaction name="actionShowMoves"/>
   <addaction name="actionShowHideMainDisplay"/>
   <addaction name="separator"/>
   <addaction name="actionPauseResume"/>
   <addaction name="actionTournamentClock"/>
  </widget>
  <action name="actionAbout_Poker_Buddy">
   <property name="text">
    <string>About Poker Buddy...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionQuickStart">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_rocket_64x64.png</normaloff>:/Resources/b_rocket_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Quick Start &amp;Tournament</string>
   </property>
   <property name="iconText">
    <string>Quick Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_restart_64x64.png</normaloff>:/Resources/b_restart_64x64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset Tournament</string>
   </property>
   <property name="iconText">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_configure_64x64.png</normaloff>:/Resources/b_configure_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Setup Tournament...</string>
   </property>
   <property name="iconText">
    <string>Setup</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAuthorize">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_auth_64x64.png</normaloff>:/Resources/b_auth_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Authorize Remote Device...</string>
   </property>
   <property name="iconText">
    <string>Authorize</string>
   </property>
  </action>
  <action name="actionPlan">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_new_64x64.png</normaloff>:/Resources/b_new_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Plan Tournament...</string>
   </property>
   <property name="iconText">
    <string>Plan</string>
   </property>
  </action>
  <action name="actionShowHideMainDisplay">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_display_64x64.png</normaloff>:/Resources/b_display_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Show / Hide Main &amp;Display</string>
   </property>
   <property name="iconText">
    <string>Display</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionShowMoves">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_move_64x64.png</normaloff>:/Resources/b_move_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Show Player Moves...</string>
   </property>
   <property name="iconText">
    <string>Moves</string>
   </property>
  </action>
  <action name="actionRebalance">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_rebalance_64x64.png</normaloff>:/Resources/b_rebalance_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Rebalance Tables...</string>
   </property>
   <property name="iconText">
    <string>Rebalance</string>
   </property>
  </action>
  <action name="actionPauseResume">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_play_pause_64x64.png</normaloff>:/Resources/b_play_pause_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Resume / Pause</string>
   </property>
   <property name="iconText">
    <string>Resume / Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionPreviousRound">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_previous_64x64.png</normaloff>:/Resources/b_previous_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Previous Round</string>
   </property>
   <property name="iconText">
    <string>Previous Round</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="actionNextRound">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_next_64x64.png</normaloff>:/Resources/b_next_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Next Round</string>
   </property>
   <property name="iconText">
    <string>Next Round</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
  <action name="actionCallClock">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_call_clock_64x64.png</normaloff>:/Resources/b_call_clock_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Call the Clock</string>
   </property>
   <property name="iconText">
    <string>Call the Clock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionEndGame">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_stop_64x64.png</normaloff>:/Resources/b_stop_64x64.png</iconset>
   </property>
   <property name="text">
    <string>End Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="TBQt.qrc">
     <normaloff>:/Resources/b_export_64x64.png</normaloff>:/Resources/b_export_64x64.png</iconset>
   </property>
   <property name="text">
    <string>Export Results...</string>
   </property>
   <property name="iconText">
    <string>Results</string>
   </property>
  </action>
  <action name="actionTournamentClock">
   <property name="text">
    <string>--:--</string>
   </property>
   <property name="toolTip">
    <string>Tournament clock</string>
   </property>
   <property name="font">
    <font>
     <family>IBM Plex Sans</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionShowHideSeatingChart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show / Hide Seating Chart</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TBQt.qrc"/>
 </resources>
 <connections/>
</ui>
